lib/modules/manager/terraform/extractors/others/providers.ts

```diff
@@ -22,7 +26,6 @@ export class ProvidersExtractor extends TerraformProviderExtractor {
             currentValue: providerTypeElement.version,
             managerData: {
               moduleName: providerTypeName,
-              source: providerTypeElement.source,
```

Comment:
was this a bug?

Reply:
This would be always `undefined` as provider blocks do not have a source field. This has been a copy and paste error.